api/api/controllers/experimental_controller.py

```diff
@@ -33,7 +33,7 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-@check_experimental_feature_value
+@deprecate_endpoint()
```

Comment:
We shouldn't remove `@check_experimental_feature_value` since it would break the endpoint if the `experimental_features` config is enabled.

That's the reason why the [test_experimental_features.py](https://github.com/wazuh/wazuh-qa/blob/master/tests/integration/test_api/test_config/test_experimental_features/test_experimental_features.py) API integration test is failing [here](https://github.com/wazuh/wazuh/pull/17908/checks?check_run_id=15014508721)

```suggestion
@deprecate_endpoint()
@check_experimental_feature_value
```

Same to all the other functions